cmd/statusd/main.go

```diff
@@ -62,6 +62,8 @@ var (
 	// Push Notification
 	enablePN     = flag.Bool("shh.notify", false, "Node is capable of sending Push Notifications")
 	firebaseAuth = flag.String("shh.firebaseauth", "", "FCM Authorization Key used for sending Push Notifications")
+
+	exitWhenSynced = flag.Int("exit-when-synced", -1, "Timeout for blockchain sync and exit in minutes, zero is infinite")
```

Comment:
That's a separate mode of an app. Maybe we should add this to the documentation.
"Special mode of the app to sync blockchain and exit when synced. Timeout is  N seconds. If N is zero, timeout is infinite"

The flag name was probably specified in the task, but I'd rather call it `-sync-and-exit`.
```bash
> statusd -sync-and-exit
```
is more readable to my eyes, IMO.

Reply:
Also, please note that the waiting time is in minutes, that is not obvious without reading code.

Reply:
Actually, `-sync-and-exit` looks better to me than `sync-only`.

Reply:
About the waiting time: `statusd -h` should print the usage line which says "in minutes".  But I can also add your description to somewhere else as well if you like. Where else do you use for denoting such things in the project?

Reply:
I'll change "Timeout for blockchain sync and exit in minutes, zero is infinite" to "Timeout in minutes for blockchain sync and exit, zero is infinite" which is more intelligible.

Reply:
any reasons not to use flag.Duration ?

Reply:
We could but I think we rely on only minutes here.